test/image/mocks/bar_attrs_group_norm.json

```diff
@@ -0,0 +1,21 @@
+{
+    "data":[
+        {
+            "base":6,
+            "y":[1,2,3,4],
```

Comment:
Yeah, I really think it should be size. `y` is the primary data "value" attribute (for vertical bars) and for a bar chart, the bar's value is its size, where it starts or ends is a secondary level of analysis.